extensions/postgres/tests/find_many/pagination/filters.rs

```diff
@@ -871,17 +956,17 @@ async fn compound_pk_implicit_order_with_nulls_and_before() {
           "edges": [
             {
               "node": {
-                "name": "Naukio",
-                "email": "meow3@example.com"
+                "name": "Musti",
+                "email": null
               },
-              "cursor": "WyJOYXVraW8iLCAibWVvdzNAZXhhbXBsZS5jb20iXQ=="
+              "cursor": "WyJNdXN0aSIsIG51bGxd"
             }
           ],
           "pageInfo": {
             "hasNextPage": true,
-            "hasPreviousPage": true,
-            "startCursor": "WyJOYXVraW8iLCAibWVvdzNAZXhhbXBsZS5jb20iXQ==",
-            "endCursor": "WyJOYXVraW8iLCAibWVvdzNAZXhhbXBsZS5jb20iXQ=="
+            "hasPreviousPage": false,
+            "startCursor": "WyJNdXN0aSIsIG51bGxd",
+            "endCursor": "WyJNdXN0aSIsIG51bGxd"
```

Comment:
So we had also a bug where both Asc and Desc were with nulls first. I don't remember why we even bother with defining the null position, it was something something complaint complaint two years ago.

Reply:
So now it's all asc nulls first, desc nulls last.